modules/javafx.graphics/src/main/native-glass/monocle/android/nativeBridge.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, 2015, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
That should be `2012, 2019, Oracle...` so as not to lose the initial year (well actually `2012, 2020, Oracle...`  as of Jan 1, but that will get updated the next time I run my copyright script, which I plan to do right before RDP2, so either 2019 or 2020 for the last-modified year is fine).